### ISSUE TITLE
[CAIA A11y Research] Cartography Team: Pilot Session - Secondary Nav (January 2024)

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#73999]
# CAIA A11y Support

This is a child ticket of the research ticket [73996](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73996) - created to track work completed for the pilot session. 

## Assignee
- Jamie

## Description

A CAIA a11y will participate as in a pilot of the Cartography team's [secondary nav research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73996).

## Team Info.
- Team name: Cartography
- OCTO-DE product owner: Patrick Bateman 
- Product manager: Wes Rowe 
- Designer: Jonathan Nelson, Florence McCafferty
- FE engineer: Jim Zaccardo
- Product/team Slack channel: mhv-on-vagov-cartography-team

## Pilot Session
- Monday, Jan. 22, 2024 at 10 a.m. 
- **Prototype or mockups:**
    - [Prototype A](https://codepen.io/figaro/project/live/AMVvEP)
    - [Prototype B](https://codepen.io/figaro/project/live/AqoeoW)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/secondary-nav/Comparison-study/conversation-guide.md)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] Determine date and time: 1/22/24, 10am
- [x] Add to a11y Capacity Calendar
- [x] Schedule CAIA a11y: @coforma-jamie
- [x] Review prototypes in CodePen: [Prototype A](https://codepen.io/figaro/project/live/AMVvEP) and [Prototype B](https://codepen.io/figaro/project/live/AqoeoW)
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73798
- [x] Review [research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/secondary-nav/Comparison-study/conversation-guide.md).
- [x] Run pilot
```
